demo/admin/package.json

```diff
@@ -17,7 +17,7 @@
         "lint:eslint": "eslint --max-warnings 0  --config ./.eslintrc.cli.js --ext .ts,.tsx,.js,.jsx,.json,.md src/ package.json",
         "lint:tsc": "tsc --project .",
         "lint:generated-files-not-modified": "$npm_execpath admin-generator && git diff --exit-code HEAD --",
-        "start": "run-s intl:compile && run-p gql:types generate-block-types && dotenv -- cross-env BABEL_ENV=development webpack serve --config ./webpack.config.ts --mode development"
+        "start": "run-s intl:compile && run-p gql:types generate-block-types && chokidar --initial  -s \"../../packages/admin/*/src/**\" -c \"kill-port 8000 && dotenv -- vite --force\""
```

Comment:
can't you use the ADMIN_PORT env var here?

Reply:
Fixed: e92a63f242db8eaf70262a34383a5468f1feb27c